server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/data/assemblers/ProgramAreaResourceAssembler.java

```diff
@@ -10,6 +10,8 @@
 import ca.bc.gov.nrs.wfprev.data.entities.ProgramAreaEntity;
 import ca.bc.gov.nrs.wfprev.data.models.ProgramAreaModel;
 
+import java.util.UUID;
```

Comment:
I think We're using strings elsewhere vs. the UUID object. Are we going to swap the rest over to use UUIDs as well?

Reply:
I meant to talk to you about this.  I think we're going to have to.  It was failing on the findById otherwise.